docs/reference/docs/reindex.asciidoc

```diff
@@ -118,8 +118,11 @@ POST _reindex
 // CONSOLE
 // TEST[setup:twitter]
 
-By default, version conflicts abort the `_reindex` process, but you can just
-count them by setting `"conflicts": "proceed"` in the request body:
+By default, version conflicts abort the `_reindex` process. The `"conflicts"` request body
+parameter can be used to instruct `_reindex` to proceed with next document on version conflicts.
```

Comment:
Suggestion:
parameter can be used to instruct `_reindex` to proceed with the next document on version conflicts.

Reply:
Fixed in 7f76ba806de6513e68c3d183e3c01f05e3a484f6